rpmlint/descriptions/PythonCheck.toml

```diff
@@ -0,0 +1,20 @@
+python-tests-in-package="""
+test/ or tests/ directory in package directory. Tests should not be included in %{python_sitelib}/<pkgname>."""
+python-doc-in-package="""
+doc/ or docs/ directory in package directory. Documentation should go into %{docdir}, not %{python_sitelib}/<pkgname>"""
+python-egg-info-distutils-style="""
+The package's egg-info is a distutils style file, rather than a setuptools
+style directory. Please convert to setuptools style.
```

Comment:
If we keep it, this needs more information. This error is not actionable. What does "convert to setuptools style" mean and how do i do it? Also, this does not explain why is this wrong.

Reply:
Note there are packages that _intentionally_ use distutils for various reasons. For example, `createrepo_c` and other `rpm-software-management` projects providing Python bindings use `distutils` so that the skeleton needed for a Python package to be represented in pip is there.